### PR TITLE
LogTalk tutorial in Bulgarian

### PR DESCRIPTION
**Pull request by vsraptor**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: bg-bg/logtalk-bg.html.markdown [hunk 1]
```diff
@@ -0,0 +1,592 @@
+---
+language: Logtalk
+contributors:
+    - ["Paulo Moura", "http://github.com/pmoura"]
+translators:
+    - ["vsraptor", "https://github.com/vsraptor"]
+lang: bg-bg
+
```
Comment (prertik) [inline bg-bg/logtalk-bg.html.markdown]: Could you follow the standard here? For reference [https://github.com/adambard/learnxinyminutes-docs/blob/master/bg-bg/perl-bg.html.markdown](https://github.com/adambard/learnxinyminutes-docs/blob/master/bg-bg/perl-bg.html.markdown).

Comment (divayprakash) [reply bg-bg/logtalk-bg.html.markdown]: @prertik What seems to be the issue here?

Comment (prertik) [reply bg-bg/logtalk-bg.html.markdown]: Well, it's missing the format. Following should be the format, taken from the reference:
```md
name: perl
category: language
language: perl
filename: learnperl-bg.pl
contributors:
    - ["Korjavin Ivan", "http://github.com/korjavin"]
    - ["Dan Book", "http://github.com/Grinnz"]
translators:
    - ["Красимир Беров", "https://github.com/kberov"]
lang: bg-bg
```
Additionally, there's no downloadable file. Why?

**Comment by prertik** [comment]
@vsraptor 
Additionally, it would be great if the commit author would merge these commits into a single commit with a good commit message. Commit message as `Add LogTalk tutorial in Bulgarian language` would be good too. Also, it would be cool if the commit would be `signed-off`.

**Comment by divayprakash** [comment]
@prertik There doesn't seem to be any movement on this. Would you be okay with merge this using the 'Squash and merge' method to combine all 8 commits into a single one? I'd be happy to make the changes to the frontmatter as you pointed out once it is merged as I cannot push to this PR.

**Comment by prertik** [comment]
Sure @divayprakash! That seems the correct way. Thanks!

**Pull request merged**